### DISCUSSION TITLE
Custom Fonts in GrapeJS Studio

### DISCUSSION THREAD

**Discussion by Jaymate123** [GrapesJS/grapesjs#5733] [Q&A]
What is the best / recommended way to add Custom fonts to GrapeJS **Studio**?

I understand studio is in alpha / very early stages, just curious if there is actually a proper way.
 
I can add fonts once I export it, but it would be nice to have them natively within the editor.

If I use the 'Import Code' button to add in fonts (or just use this feature in general/at all) my site breaks - the JS from the 'Plugins' I have installed stops working, and CSS completely disappears from all pages other than the Home page.

**Comment by artf** [comment]
Yeah, the Studio at the moment is missing that, so I'd only suggest being patient for now, definitely will be added one day.
